### DISCUSSION TITLE
Support for transitiveRemediation on npm7 (lockFileVersion 2)

### DISCUSSION THREAD

**Discussion by jmac105** [renovatebot/renovate#10370] [Suggest an Idea]
Hi,

I see that this feature was briefly merged in #9402 before being reverted later that day. I was wondering if you're able to provide a timeline for when v2 lockfiles will support transitiveRemediation? 

Thanks

**Comment by rarkins** [comment]
No, unfortunately not. I created #10371 for tracking purposes
